tests/common/videolayerrenderer.cpp

```diff
@@ -45,6 +46,155 @@ bool VideoLayerRenderer::Init(uint32_t width, uint32_t height, uint32_t format,
   return true;
 }
 
+static int get_bpp_from_format(uint32_t format, size_t plane)
+{
+	//assert(plane < drv_num_planes_from_format(format));
+	switch (format) {
+	  case DRM_FORMAT_BGR233:
+	  case DRM_FORMAT_C8:
```

Comment:
Fix indentation, git diff | clang-format-diff-3.5 -p 1 -style=file  should do it before commiting the changes